docker/keeper/Dockerfile

```diff
@@ -71,20 +74,20 @@ RUN arch=${TARGETARCH:-amd64} \
         fi \
     ; done \
     && rm /tmp/*.tgz /install -r \
-    && addgroup -S -g 101 clickhouse \
-    && adduser -S -h /var/lib/clickhouse -s /bin/bash -G clickhouse -g "ClickHouse keeper" -u 101 clickhouse \
-    && mkdir -p /var/lib/clickhouse /var/log/clickhouse-keeper /etc/clickhouse-keeper \
-    && chown clickhouse:clickhouse /var/lib/clickhouse \
-    && chown root:clickhouse /var/log/clickhouse-keeper \
```

Comment:
This type of change must be the same with `docker/server/Dockerfile.alpine` image.

Reply:
Oh. I didn't get that you also wanted the server image changed to match, but it makes sense. Done.